docs/resources/mnq_credential.md

```diff
@@ -0,0 +1,88 @@
+---
+page_title: "Scaleway: scaleway_mnq_credential"
+description: |-
+Manages Scaleway Messaging and Queuing Credential.
+---
+
+# scaleway_mnq_namespace
+
+This Terraform configuration creates and manage a Scaleway MNQ credential associated with a namespace.
+For additional details, kindly refer to our [website](https://www.scaleway.com/en/docs/serverless/messaging/)
+and the [godoc](https://pkg.go.dev/github.com/scaleway/scaleway-sdk-go@master/api/mnq/v1alpha1#pkg-index).
+
+## Examples
+
+### NATS credential
+
+```hcl
+resource "scaleway_mnq_namespace" "main" {
+  name     = "mnq-ns"
+  protocol = "nats"
+  region   = "fr-par"
```

Comment:
This example has `region` but the SNS example doesn't

Reply:
I would suggest removing it completly